src/lib.rs

```diff
@@ -25,7 +27,9 @@
 //! * `atmega32u4`
 //! * `atmega48p`
 //! * `atmega64`
+//! * `attiny841`
```

Comment:
Here the `attiny84` is missing (already on the attiny84's original commit).

Reply:
It seems like atmega644 is also missing

Reply:
Ah, I'll add those after merging this PR.